### PR TITLE
Use common memcached.GetMemcachedServerListWithInetString()

### PR DESCRIPTION
**Pull request by stuggi**
Use common func to retrieve the memcached server list for a memcached instance.

### CODE REVIEW

File: controllers/autoscaling_controller.go [hunk 1]
```diff
@@ -345,7 +344,7 @@ func (r *AutoscalingReconciler) reconcileNormal(
 	//
 	// Check for required memcached used for caching
 	//
-	memcached, err := r.getAutoscalingMemcached(ctx, helper, instance)
```
Comment (vyzigold) [inline controllers/autoscaling_controller.go:348]: I think the `getAutoscalingMemcached` function on line 602 can now be deleted.

**Review by jlarriba — APPROVED** [review]

**Comment by stuggi** [comment]
rebased

**Pull request merged**